test/presubmit-tests.sh

```diff
@@ -108,6 +108,28 @@ function initialize_environment() {
   readonly IS_PRESUBMIT_EXEMPT_PR
 }
 
+function get_node() {
+  echo "Script is running as $(whoami) on $(hostname) and directory structure is $(find .)"
+  # It's Stretch and https://github.com/tektoncd/dashboard/blob/master/package.json 
+  # denotes the Node.js and npm versions
+  apt-get update
+  apt-get install -y curl
+  curl -O https://nodejs.org/dist/v10.15.3/node-v10.15.3-linux-x64.tar.xz
+  tar xf node-v10.15.3-linux-x64.tar.xz
+  export PATH=$PATH:$(pwd)/node-v10.15.3-linux-x64/bin
+}
+
+function node_test() {
+  echo "Running node tests from $(pwd)"
+  mkdir ~/.npm-global
+  npm config set prefix '~/.npm-global'
+  export PATH=$PATH:$HOME/.npm-global/bin
+  npm install -g --unsafe-perm node-sass
```

Comment:
We shouldn't need to install any packages globally, and node-sass is already specified in package.json so seems a bit odd to have this. Were you hitting a particular issue without it?

Reply:
In an old commit yeah, then I added the npm config fun stuff (it was permission denied to do with node_modules folder and things running as `nobody` presumably)

Just removed it and all is well, have a look at https://prow.tekton.dev/log?job=pull-tekton-dashboard-tests&id=1125687504725348352 😄 (the latest Prow run for this PR)